ui/index.html

```diff
@@ -4,11 +4,9 @@
         <meta charset="utf-8">
         <title ng-bind="getWindowTitle()">Treeherder</title>
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-        <link id="favicon" type="image/png" rel="shortcut icon" href="img/tree_open.png">
-        <th-favicon-link></th-favicon-link>
```

Comment:
Removing the custom `<th-favicon-link>` fixes [bug 1143255](https://bugzilla.mozilla.org/show_bug.cgi?id=1143255) too :-)